ra/ra.go

```diff
@@ -176,13 +176,30 @@ func problemIsTimeout(err error) bool {
 	return false
 }
 
+// forbiddenMailDomains is a map of domain names we do not allow after the
+// @ symbol in contact mailto addresses. These are frequently used when
+// copy-pasting example configurations and would not result in expiration
+// messages and subscriber communications reaching the user that created the
+// registration if allowed.
+var forbiddenMailDomains = map[string]bool{
+	// https://tools.ietf.org/html/rfc2606#section-3
+	"example.com": true,
+	"example.net": true,
+	"example.org": true,
+}
+
 func validateEmail(ctx context.Context, address string, resolver bdns.DNSClient) error {
 	email, err := mail.ParseAddress(address)
 	if err != nil {
 		return unparseableEmailError
 	}
 	splitEmail := strings.SplitN(email.Address, "@", -1)
 	domain := strings.ToLower(splitEmail[len(splitEmail)-1])
+	if _, present := forbiddenMailDomains[domain]; present {
```

Comment:
You could simplify this as:

`if forbiddenMailDomains[domain] {`

Since the map returns a bool with a zero value of false.

Reply:
@4a6f656c Good point! Thanks, fixed in bbf7e68